dapps/frenemies/src/routes/Connect.tsx

```diff
@@ -23,14 +47,38 @@ export function Connect() {
           Welcome to Sui Frenemies game
```

Comment:
should this be `SUI` ?

Reply:
I don't think so? I think we only use `SUI` for the token.